spec/rubocop/cli_spec.rb

```diff
@@ -1627,6 +1627,23 @@ def abs(path)
       end
     end
 
+    describe '-e/--extra-details' do
+      it 'shows extra details' do
```

Comment:
excellent - that seems to work exactly as you suggest and has allowed me to remove that silly 'no details set'